packages/features/ee/workflows/components/AddVariablesDropdown.tsx

```diff
@@ -1,58 +1,63 @@
-/**
```

Comment:
All diffs in this page are because the v1 version had the same name, and I moved v2 to the same place. So this is actually the difference between v1 and v2.